packages/web-components/src/components/rux-dialog/rux-dialog.tsx

```diff
@@ -77,6 +78,11 @@ export class RuxDialog {
     })
     ruxDialogClosed!: EventEmitter<boolean | null>
 
+    @Method()
+    async toggleOpen() {
+        this.open = !this.open
+    }
+
```

Comment:
Is there any benefit to returning `this.open`, or should this just remain a void function?

Reply:
Probably can stay void. Can get this.open from the open attribute on the dialog